igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildCache.groovy

```diff
@@ -38,16 +38,18 @@ class JenkinsCache {
     @Value('${spinnaker.jedis.prefix:igor}')
     String prefix
 
+    String id = 'builds'
+
     List<String> getJobNames(String master) {
         Jedis resource = jedisPool.resource
-        List<String> jobs = resource.keys("${prefix}:${master}:*").collect { extractJobName(it) }.sort()
+        List<String> jobs = resource.keys("${prefix}:${id}:${master}:*").collect { extractJobName(it) }.sort()
```

Comment:
This addresses the keys. 

Would rather see two buildCache instances with prefix for jenkins being "${prefix}" and prefix for travis being "${prefix}:travis". 

Cannot justify adding this breaking change to the keys for existing deployments specially when it could mean every pipeline being triggered and losing the ability to red/black or rollback